app/app.iml~

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Delete this whole file, because it is a backup file we do not need to store on GitHub. Also if you can, I would add it to our .gitignore file so this doesn't happen again.